src/App.css

```diff
@@ -5,6 +5,7 @@
   --secondary-color: #fff;
   --txt-color: #424b54;
   --border-color: rgba(148, 161, 179, 0.5);
+  --danger-color: #f14b4b;
```

Comment:
Try using either color code or RGB for consistency.

Reply:
ok